packages/nimble-components/src/rich-text/editor/tests/rich-text-editor.spec.ts

```diff
@@ -642,6 +642,186 @@ describe('RichTextEditor', () => {
                 'P'
             ]);
         });
+
+        describe('Absolute link interactions in the editor', () => {
+            describe('various absolute links without other nodes and marks', () => {
```

Comment:
Could you provide some background about how you decided which tests go in the element spec files, which go in the model spec files, and which go in both? The philosophy would be helpful to add to the Testing section of the PR description.

I haven't gone through thoroughly but I get a sense that we might be testing some functionality at the element level which could be tested at the model level and that we might be duplicating some cases between both. Ideally we'd be covering cases at the lowest possible level (model) and not duplicating cases, but I recognize that won't always be possible.

Reply:
I have updated the `testing` section of the PR description briefly about the approach followed in each file. Please let us know if you need more details or something we should consider changing in it.

Reply:
Thanks, that description is helpful.